packages/camera/android/src/main/java/io/flutter/plugins/camera/CameraPlugin.java

```diff
@@ -471,6 +471,7 @@ public void onOpened(@NonNull CameraDevice cameraDevice) {
                     startPreview();
                   } catch (CameraAccessException e) {
                     if (result != null) result.error("CameraAccess", e.getMessage(), null);
+                    return;
```

Comment:
Looking at this again, I wonder if we should also close the camera here?

Reply:
Closing camera now